internal/assets/dockerfiles/java-war-tomcat/Dockerfile

```diff
@@ -0,0 +1,26 @@
+#   Copyright IBM Corporation 2020
+#
+#   Licensed under the Apache License, Version 2.0 (the "License");
+#   you may not use this file except in compliance with the License.
+#   You may obtain a copy of the License at
+#
+#        http://www.apache.org/licenses/LICENSE-2.0
+#
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS,
+#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#   See the License for the specific language governing permissions and
+#   limitations under the License.
+
+FROM registry.access.redhat.com/ubi8/ubi:8.3-199 AS install_tomcat
+RUN dnf update && dnf install -y java-11-openjdk-devel wget
+WORKDIR /usr/local
+RUN wget https://mirrors.estointernet.in/apache/tomcat/tomcat-9/v9.0.39/bin/apache-tomcat-9.0.39.tar.gz && tar -zxf apache-tomcat-9.0.39.tar.gz && mv apache-tomcat-9.0.39 tomcat9
```

Comment:
Would https://downloads.apache.org/tomcat/tomcat-9/v9.0.39/bin/apache-tomcat-9.0.39.tar.gz link be better, since it does not specify a geo?